packages/blockchain-wallet-v4-frontend/src/hooks/useApplePay/utils/isApplePayAvailable/isApplePayAvailable.ts

```diff
@@ -1,2 +1,11 @@
-export const isApplePayAvailable = (): boolean =>
-  (window.ApplePaySession && ApplePaySession?.canMakePayments()) || false
+export const isApplePayAvailable = (): boolean => {
+  if (!window.ApplePaySession) return false
+
+  try {
+    return ApplePaySession?.canMakePayments()
```

Comment:
you don't need this question mark here, you already checked it on the top

Reply:
ok, I removed the ?